Cargo.toml

```diff
@@ -20,22 +20,21 @@ include = [
 ]
 
 [dependencies]
-bytemuck = "1.12"
-raw-window-handle = "0.4"
-thiserror = "1.0"
-ultraviolet = "0.9"
-wgpu = "0.13"
+bytemuck = "1.12.1"
+raw-window-handle = "0.5.0"
+thiserror = "1.0.37"
+ultraviolet = "0.9.0"
```

Comment:
I would prefer to remove all of the patch numbers. They aren't necessary and the only time they could potentially matter is in the rare case that the current patch version you are using is yanked and a newer patch version is not released to fix whatever caused the yank.

I would even go as far to remove the minor numbers from any version that is `>= 1`. That's what I have been doing in other crates that I've worked on recently.